stories/Select/Guideline.mdx

```diff
@@ -0,0 +1,74 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  Primary,
+  Controls,
+  Unstyled,
+  Source,
+} from '@storybook/blocks';
+import { Select } from '../../src/lib/components/selectv2/Selectv2.component';
+
+import * as SelectStories from './selectv2.stories';
+
+<Meta title="Guideline" of={SelectStories} />
+
+# Select
```

Comment:
I think we are missing some guidelines on when/how to use icons for the options ?

Reply:
From what I remember, we don't use icons for statuses (unlike what is displayed in the story), it's more _descriptive_ icons that help the user differentiates the different options (for instance, different kinds of workflows).
And if such icons are present in front of labels, they should be present for _all_ labels.

Reply:
The "selected" status visual information doesn't come with the _checked" icon, but with a "selected" color.